packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -118,10 +119,15 @@ class BaseCacheManager {
       return;
     }
 
-    let response = await fetch(precacheEntry.getNetworkRequest(), {
+    let response = await this._requestWrapper.fetch({
```

Comment:
Have you considered using [`RequestWrapper.fetchAndCache`](https://github.com/GoogleChrome/sw-helpers/blob/master/packages/sw-runtime-caching/src/lib/request-wrapper.js#L177) here instead of `RequestWrapper.fetch`? All the "value add" stuff is in `RequestWrapper.fetchAndCache`.